src/boot/efi/stub.c

```diff
@@ -474,6 +572,16 @@ static EFI_STATUS load_addons(
                                 .filename =  xstrdup16(items[i]),
                         };
                 }
+
+                if (ucode_addons && PE_SECTION_VECTOR_IS_SET(sections + UNIFIED_SECTION_UCODE)) {
```

Comment:
mm, i think we should probably apply the uname conditioning above only to cmdline+dtb, but not to the ucode part, because ucode really shouldn't have an efefct on kernel compatibility.

Reply:
hmm sure, I can change that - but it makes things a bit more complex and inconsistent if some addons behave differently?

Reply:
Maybe: Ucode updates are sometimes rolled out for CPU mitigations (spectre/meltdown and friends). Those often go hand in hand with kernel mitigations, maybe that is a reason why someone would want to tie these two together?